params/bamford_buzzard.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 2.5
+    
+# e.g. "anas_krystallinus"    
+species.name <- "bamford_buzzard"
```

Comment:
Thanks @bobturneruk for the contribution, we've had a few submissions of Bamford buzzards recently, would you mind changing the `species.name` please?

Reply:
Good spot, I've renamed the species.